transaction.rb

```diff
@@ -0,0 +1,30 @@
+require_relative "customer"
+require_relative "teller"
+
+class Transaction
+  attr_accessor :transaction_type, :customer, :teller, :transaction_amount, :account_type
```

Comment:
This being the in the `Transaction` class, just `type` and `amount` should do as attributes. I think pretending both with `transaction` is redundant.

Also I'm too sure about having an `account_type` on the Transaction. what's the rationale behind it.
why not the account object itself or the account number?

Reply:
the customer may have savings and a maybe a current account so I am taking the account type to know which particular account of the customer  the transaction is being performed on

Reply:
> the customer may have savings and a maybe a current account so I am taking the account type to know which particular account of the customer the transaction is being performed on and also its shown in my return message the type of account that the transaction was done on or I should limit my model to the customer having only one account